src/lib/support/jsontlv/JsonToTlv.cpp

```diff
@@ -466,4 +466,29 @@ CHIP_ERROR ConvertTlvTag(const uint64_t tagNumber, TLV::Tag & tag)
 {
     return InternalConvertTlvTag(tagNumber, tag);
 }
+
+CHIP_ERROR SingleAttributeJsonToTlv(const std::string & jsonString, MutableByteSpan & tlv)
+{
+    Platform::ScopedMemoryBufferWithSize<uint8_t> buf;
+    VerifyOrReturnError(buf.Calloc(tlv.size()), CHIP_ERROR_NO_MEMORY);
+    MutableByteSpan dataWithStruct(buf.Get(), buf.AllocatedSize());
+    ReturnErrorOnFailure(JsonToTlv(jsonString, dataWithStruct));
```

Comment:
So this is assuming the provided buffer is large enough to hold the value _and all the struct framing_.

This seems like a bad assumption that we should not make.  The actual size we should use for our temp buffer should be something more like `TLV::EstimateStructOverhead(tlv.size())`, no?